### PR TITLE
Remove bootstrap from signup/login components

### PR DESCRIPTION
**Pull request by clari182**
- Helps with #1732 

Pages:
[/login](https://deploy-preview-1816--staging-aiid.netlify.app/login)
[/signup](https://deploy-preview-1816--staging-aiid.netlify.app/signup)
[/forgotpassword](https://deploy-preview-1816--staging-aiid.netlify.app/forgotpassword)
[/resetpassword](https://deploy-preview-1816--staging-aiid.netlify.app/resetpassword)

### CODE REVIEW

File: site/gatsby-site/src/pages/forgotpassword.js [hunk 1]
```diff
@@ -47,26 +47,33 @@ const ForgotPassword = (props) => {
...
+          values,
+          errors,
+          touched,
+          handleChange,
+          handleBlur,
+          handleSubmit,
+          isSubmitting,
+          isValid,
+        }) => (
+          <div>
```
Comment (pdcp1) [inline site/gatsby-site/src/pages/forgotpassword.js:60]: Can we remove this `<div>`?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

**Review by pdcp1 — APPROVED** [review]

**Comment by clari182** [comment]
Hey @pdcp1 can you do another review? I found some hydration issues while testing and fixed them (hopefully).

File: site/gatsby-site/src/components/forms/TextInputGroup.js [hunk 2]
```diff
@@ -122,24 +132,34 @@ const Input = ({
...
-      value={values[name] || props.defaultValue || ''}
-      {...props}
-      onChange={handleChange}
-      onBlur={handleBlur}
-    />
-    {addOnComponent}
-  </div>
-);
+}) => {
+  const [inputName, setInputName] = useState('');
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/forms/TextInputGroup.js:136]: This adds an extra render on every input. Why do you need it?

Comment (clari182) [reply site/gatsby-site/src/components/forms/TextInputGroup.js]: Fixed!

**Pull request merged**